src/coreclr/jit/importer.cpp

```diff
@@ -21359,35 +21251,12 @@ void Compiler::impDevirtualizeCall(GenTreeCall*            call,
                                 //
                                 JITDUMP("Success! invoking unboxed entry point on local copy, and passing method table "
                                         "arg\n");
-                                call->gtCallThisArg = gtNewCallArgs(localCopyThis);
+                                // TODO-CallArgs-REVIEW: Might discard commas otherwise?
+                                assert(thisObj == thisArg->GetEarlyNode());
+                                thisArg->SetEarlyNode(localCopyThis);
```

Comment:
@AndyAyersMS Just FYI: looks like there are a few places here that are missing some handling for commas. I added the assert for now but can open an issue for it.